src/convertor.ts

```diff
@@ -185,6 +192,116 @@ export class PrismaConvertor {
 		return decorator
 	}
 
+	extractValidationDecoratorsFromField = (
+		dmmfField: DMMF.Field,
+	): DecoratorComponent[] => {
+		const importFrom = 'class-validator'
+		const decorators: DecoratorComponent[] = []
+
+		if (dmmfField.isList) {
+			const decorator = new DecoratorComponent({
+				name: 'IsArray',
+				importFrom,
+			})
+			decorators.push(decorator)
+		}
+
+		if (!dmmfField.isRequired) {
+			const decorator = new DecoratorComponent({
+				name: 'IsOptional',
+				importFrom,
+			})
+			decorators.push(decorator)
+		}
+
+		const decoratorArgs: DecoratorArgs = {
+			name: undefined,
+			importFrom,
+			params: dmmfField.isList ? { each: true } : undefined,
+		}
+
+		if (
+			dmmfField.type &&
+			dmmfField.type !== 'any' &&
+			dmmfField.kind !== 'enum'
+		) {
+			switch (dmmfField.type) {
+				case 'String':
+					decoratorArgs.name = 'IsString'
+					break
+				case 'Number':
+					decoratorArgs.name = 'IsNumber'
+					break
+				case 'Decimal':
+					decoratorArgs.name = 'IsDecimal'
+					break
+				case 'Float':
+					decoratorArgs.name = 'IsDecimal'
+					break
+				case 'Object':
+					decoratorArgs.name = 'IsObject'
+					break
+				case 'Json':
+					decoratorArgs.name = 'IsJSON'
+					break
+				case 'Boolean':
+					decoratorArgs.name = 'IsBoolean'
+					break
+				case 'Date':
+					decoratorArgs.name = 'IsDate'
+					break
+				case 'DateTime':
+					decoratorArgs.name = 'IsDate'
+					break
+				case 'Int':
+					decoratorArgs.name = 'IsInt'
+					break
+				case 'BigInt':
+					decoratorArgs.name = 'IsInt'
+					break
+				case 'SmallInt':
+					decoratorArgs.name = 'IsInt'
+					break
```

Comment:
```suggestion
				case 'Int':
				case 'BigInt':
				case 'SmallInt':
					decoratorArgs.name = 'IsInt'
					break
```

you could use use a fall-through here, making it easier to read. Otherwise a fixed map at the top of the file would also be better to not have this huge switch-case here, have a look at the `primitiveMapType` variable